doc/WinGetConfigure.md

```diff
@@ -0,0 +1,210 @@
+# WinGet and Desired State Configuration
+
+The `winget configure` command (available as an experimental feature in WinGet 1.5-preview(3) and later) is designed to streamline the process of getting your Windows environment to a desired state. WinGet is leveraging PowerShell Core (not Windows PowerShell) and PowerShell DSC (Desired State Configuration) to achieve this goal. WinGet is introducing a new YAML formatted file to define the desired state in a declarative manner rather than an imperative manner. Our goal is to incorporate this new file and it's syntax into PowerShell DSC v3 in the future.
```

Comment:
```suggestion
The `winget configure` command (available as an experimental feature in WinGet 1.5-preview(3) and later) is designed to streamline the process of getting your Windows environment to a desired state. WinGet is leveraging PowerShell 7 (not Windows PowerShell) and PowerShell DSC (Desired State Configuration) to achieve this goal. WinGet is introducing a new YAML formatted file to define the desired state in a declarative manner rather than an imperative manner. Our goal is to incorporate this new file and it's syntax into PowerShell DSC v3 in the future.
```

We don't call it `PowerShell Core` anymore